packages/nextjs/app/page.tsx

```diff
@@ -1,16 +1,26 @@
+"use client";
```

Comment:
Maybe adding a comment here to know that this line is to be able to use useAccount

Reply:
it needed not only for `useAccount`, but for any react hook